deepchem/models/tests/test_molgan_model.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+
+from deepchem.models import BasicMolGANModel as MolGAN
+
+
+class test_molgan_model(unittest.TestCase):
+  """
+  Unit testing for MolGAN basic layers
+  """
+
+  def __init__(self, *args, **kwargs):
```

Comment:
You should use a `setup()` method rather than a constructor for setting these.  That way it will be executed before each test, and you don't have to worry about information leaking between tests.

Reply:
could you show an example, so I understand what you mean?

Reply:
@peastman I think I now understand what you mean, but to be sure, do you meant [this](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp)?

Reply:
Right.  Here's an example from an existing test:

https://github.com/deepchem/deepchem/blob/c68f7bab307599f159a8896b63a7e7e39e46725c/deepchem/models/tests/test_pretrained_keras.py#L38-L46